### PR TITLE
EVG-12929 Create uservoice feedback component

### PR DESCRIPTION
**Pull request by khelif96**
This adds a feature requests link to the bottom of all pages that takes the user to uservoice. 
A new `.cmdrc.json` is needed to test
image

Depends on https://github.com/evergreen-ci/evergreen/pull/3995

### CODE REVIEW

**Comment by SupaJoon** [comment]
Screen Shot 2020-09-16 at 9 45 47 AM
I think it's inconvenient that the feedback label covers a patch name. A solution I can think of is to create a static footer that spans the the bottom of the page and have rest of the page contents rendered above it

**Review by SupaJoon — CHANGES_REQUESTED** [review]
I think it's inconvenient that the label covers a patch name. A solution that could work is to render the label in a footer and have all of the other page contents render above.

Screen Shot 2020-09-16 at 9 45 47 AM

**Comment by bsamek** [comment]
Would it be better to put this at the bottom right? That tends to be the least information dense part of the page.

File: src/components/Feedback/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import Icon from "components/icons/Icon";
+import { StyledLink } from "components/styles";
+import { getUserVoiceKey } from "utils/getEnvironmentVariables";
+
+const { green } = uiColors;
+export const Feedback: React.FC = () => {
+  const userVoiceUrl = `https://feedback.mongodb.com/forums/${getUserVoiceKey()}`;
```
[the remark this answers is not in the document]
Comment (khelif96) [reply src/components/Feedback/index.tsx:10]: https://github.com/evergreen-ci/evergreen/pull/3995/files

**Review by john-m-liu — APPROVED** [review]

**Review by SupaJoon — APPROVED** [review]

**Pull request merged**